src/utils/Navigate.ts

```diff
@@ -1,14 +1,14 @@
 import { createPinia, setActivePinia } from 'pinia'
 import { useStore } from '@/store/store'
 
-setActivePinia(createPinia())
```

Comment:
This was causing lots of unit tests to fail!

![unit tests failing](https://github.com/bcgov/business-create-ui/assets/122301442/6eee2b2a-55a1-4367-8d5e-5db68e5b691f)

I just moved it inside the function.

Reply:
This works for me before your changes. What do you think caused the tests to start failing?

Reply:
Yes, you're right. I ran the tests in main, they work. This is weird because I literally didn't touch this file whatsoever. Looking into it...

Reply:
Thanks for looking into it. It's usually worth the time to understand why something is happening.

Reply:
Interesting. So what was causing the tests to fail was line 73 in store.ts:
```typescript
import { GetFeatureFlag } from '@/utils'
```
If I believe correctly, it's since we're grabbing from the index.ts file and since Navigate is there, the store doesn't exist yet at that point. 

The way I went on about this is, I reverted this change and imported the `GetFeatureFlag` function directly from the file:
```typescript
import { GetFeatureFlag } from '../utils/feature-flag-utils'
```

Please let me know if you don't like this Sev.

Reply:
Good find.

What do you think about `import GetFeatureFlags from '@utils/feature-flag-utils.ts'`?

Sorry, typing this on my phone.

Reply:
Yup, done 👍 Also, updated the unit tests as well.